featurehub_repository.py

```diff
@@ -41,3 +41,6 @@ def __update_feature_state(self, feature_state):
 
     def is_ready(self):
         return self.ready
+
+    def feature(self, name) -> FeatureStateBaseHolder:
+        return self.features.get(name)
```

Comment:
yeah, so if I went feature("SEB") and there was no feature("SEB") this would give None? So would your code blow up?

Reply:
added tests